package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "react-dev": "webpack -d --watch",
-    "start": "node server/index.js",
+    "start": "nodemon server/index.js",
     "start:debug": "nodemon --inspect server/index.js",
```

Comment:
Mike added a start:debug command for nodemon. Let's keep npm start as without nodemon?